### PR TITLE
Added Bisulfite QC sub-workflow with QC tools

### PR DESCRIPTION
**Pull request by saimukund20**

### CODE REVIEW

File: definitions/subworkflows/Bisulfite_QC.cwl [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+           type: string?
+           default: bisulfite_QC
+
+outputs:
+    QC_directory:
+        type: Directory
+        outputSource: gather_to_sub_directory/gathered_directory
+
+steps:
+      Tool_bisulfiteconversion:
```
[the remark this answers is not in the document]
Comment (tmooney) [reply definitions/subworkflows/Bisulfite_QC.cwl]: I would avoid putting generic terms like `Tool` or `.cwl` in the names of steps.  Keep it specific to what it is.

File: definitions/tools/gather_to_sub_directory.cwl [hunk 2]
```diff
@@ -0,0 +1,41 @@
+#! /usr/bin/env cwl-runner
+
+cwlVersion: v1.0
+class: CommandLineTool
+baseCommand: ["/bin/bash","directory_gatherer.sh"]
+
+requirements:
+    - class: ShellCommandRequirement
```
Comment (tmooney) [inline definitions/tools/gather_to_sub_directory.cwl]: Is this tool using the `ShellCommandRequirement`?  It doesn't look like it's taking advantage of any shell features to invoke the script.

File: definitions/pipelines/bisulfite.cwl [hunk 3]
```diff
@@ -71,6 +76,15 @@ steps:
             reference: reference_index
         out:
             [vcf]
+    bisulfite_qc:
+        run: ../subworkflows/Bisulfite_QC.cwl
```
[the remark this answers is not in the document]
Comment (saimukund20) [reply definitions/pipelines/bisulfite.cwl]: made the changed and did a git push

File: definitions/subworkflows/Bisulfite_QC.cwl (hunk 1 above)
[the remark this answers is not in the document]
Comment (saimukund20) [reply definitions/subworkflows/Bisulfite_QC.cwl]: made the changed and did a git push

**Comment by chrisamiller** [comment]
+1 looks good to me

**Pull request merged**